bidwire/bidwire_settings.py

```diff
@@ -58,7 +66,12 @@ def get_recipients_list(env_var_name, default_recipients='bidwire-logs@googlegro
     Bid.Site.CITYOFBOSTON: {
         'scraper': CityOfBostonScraper(),
         'notifier': CityOfBostonNotifier(),
-        'recipients': EMAIL_RECIPIENTS
+        'recipients': BOSTON_BID_RECIPIENTS
```

Comment:
Please revert the change to this line. I appreciate the idea of unifying how all the scrapers are configured, but adding this means that on deploy I will have to also be modifying the configuration for `CITYOFBOSTON` scraping, which is unnecessary and adds more likelihood of screwing something that is already working fine.